docs/ibc/overview.md

```diff
@@ -1,180 +1,150 @@
-<!--
-order: 1
--->
+<!-- order: 1 -->
 
-# Overview
+ # IBC Overview
 
-Learn what IBC is, its components and use cases. {synopsis}
+Learn what IBC is, its components, and use cases. {synopsis}
 
-## What is the Interblockchain Communication Protocol (IBC)?
+## What is the Inter-Blockchain Communication Protocol (IBC)?
 
-This document serves as a guide for developers who want to write their own Inter-blockchain
-Communication Protocol (IBC) applications for custom [use-cases](https://github.com/cosmos/ics/blob/master/ibc/4_IBC_USECASES.md).
+This document serves as a guide for developers who want to write their own IBC apps for custom use cases. <!--[use-cases](https://github.com/cosmos/ics/blob/master/ibc/4_IBC_USECASES.md)-->.
```

Comment:
this link does not resolve for me, do we have use cases documented somewhere? if not, let's comment out